pkg/lib/cockpit-components-inline-notification.jsx

```diff
@@ -87,9 +87,9 @@ InlineNotification.propTypes = {
     type: PropTypes.string,
 };
 
-export const ModalError = ({ dialogError, dialogErrorDetail, id, isExpandable }) => {
+export const ModalError = ({ dialogError, dialogErrorDetail, id, isExpandable, variant }) => {
```

Comment:
Since ModalError is a bit biased already that's an error, an the use cases of showing modal error as warning is rather limited, I would suggest to not change htis here and instead use an inline alert directly in the original PR.

Reply:
Do you mean to use `InlineNotification` component or PF's `Alert` component in the original PR?

Reply:
Yes.